Content.Server/Silicons/Laws/SiliconLawEui.cs

```diff
@@ -52,8 +52,12 @@ public override void HandleMessage(EuiMessageBase msg)
             return;
 
         var player = _entityManager.GetEntity(message.Target);
+        var playerProviderComp = _entityManager.GetComponentOrNull<SiliconLawProviderComponent>(player);
 
         _siliconLawSystem.SetLaws(message.Laws, player);
+
+        if (playerProviderComp != null)
```

Comment:
```suggestion
        if (TryComp<SiliconLawProviderComponent>(player, out var playerProviderComp))
```
use TryComp for this